src/Travel-agent.js

```diff
@@ -1,28 +1,61 @@
 import User from './User';
+var moment = require('moment');
 
 class TravelAgent extends User {
   constructor() {
     super();
```

Comment:
user, travel-agent,traveler all have this.id.  You might want to add these to your super() and have these live on the user class.

Reply:
Great point. I think that was my initial intention, and somehow missed actually adding it to the super somewhere along the way.